guides/common/modules/proc_installing-openscap-plugin.adoc

```diff
@@ -19,12 +19,14 @@ The OpenSCAP plug-in consists of the main OpenSCAP plug-in itself, the OpenSCAP
 ----
 +
 Perform this command on both your {ProjectServer} and any attached {SmartProxies}.
+ifndef::satellite[]
 . Optional: Install the OpenSCAP Hammer CLI plug-in:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} --enable-foreman-cli-openscap
 ----
+endif::[]
 . Install the OpenSCAP plug-in Puppet module:
```

Comment:
To simplify this in future releases I opened https://github.com/theforeman/puppet-foreman_proxy/pull/781 so it's handled by the installer.

Reply:
@ekohl 
I am not sure, what action item for this is expected from me. All the other things are fixed in this PR except this.

Reply:
With Foreman 3.5 you'll be able to pass `--foreman-proxy-plugin-openscap-puppet-module true` instead of a manual package installation. So in the short term nothing, but a follow up PR should be created to use that.

Reply:
ok. So resolving it for now.